### ISSUE TITLE
nix-review without sandbox?

### ISSUE THREAD

**Issue by xbreak** [Mic92/nixpkgs-review#55]
I run nix as a single-user, non-root on CentOS 7.4, without sandboxing (c.f. NixOS/nix#2632), which normally works fine. But when I'm using nix-review I get errors like this:

```
[0/19 built, 0/23 copied (0.0/369.1 MiB), 0.0/81.2 MiB DL] error: cloning builder process: Invalid argument
```
When building the same package directly with `nix-build` I don't get this problems.

I suspect it might have something to do with sandboxing. If true, would it make sense to provide an option to not sandbox?

**Comment by Mic92** [comment]
Yes it would make sense to provide an option but having sandboxes enabled by default.

This is where it gets enabled: https://github.com/Mic92/nix-review/blob/3c54a9c1c32e577dad59fe19c64beae7c2ad91e8/nix_review/nix.py#L141

**Comment by risicle** [comment]
Many darwin packages don't build with sandbox enabled, this leaves `nixpkgs-review`-users unable to review a package.

**Comment by SuperSandro2000** [comment]
@risicle that should be fixed by https://github.com/Mic92/nixpkgs-review/commit/aa9cc2f5e9199736af5167f8c3f406018b6a14e7
